### PR TITLE
adding recipe doc for testing ava with selenium

### PR DESCRIPTION
**Pull request by ganeshmani**
Hi, 
This pull request closes Issue[#1023](https://github.com/avajs/ava/issues/1023). It has the recipe(docs) for testing AVA with selenium.

### CODE REVIEW

**Comment by novemberborn** [comment]
Thanks for the PR @ganeshmani!

Is there existing documentation we can link to when it comes to installing Selenium and `webdriverio` and starting it before you run tests? I'd like to keep this recipe focused on writing tests that use `webdriverio` without the other boilerplate.

Could you add examples of actual tests? Is each `client` a single browser instance, so that 
tests should run serially? Do you need to reset the client between tests? Does it make sense to use multiple clients?

**Comment by CrispusDH** [comment]
Also, webdriverio it is Not selenium binding, it implements jsonwp and w3c protocols by itself.

File: docs/recipes/testing-with-selenium.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+01:21:33.252 INFO [GridLauncherV3.lambda$buildLaunchers$3] - Launching a standalone Selenium Server on port 4444
+2019-08-26 01:21:33.291:INFO::main: Logging initialized @224ms to org.seleniumhq.jetty9.util.log.StdErrLog
+01:21:33.457 INFO [WebDriverServlet.<init>] - Initialising WebDriverServlet
+01:21:33.847 INFO [SeleniumServer.boot] - Selenium Server is up and running on port 4444
+Selenium started
+```
+To Run the testcases, it should contain the webdriver config such as browser name and other [configurations](https://github.com/webdriverio/webdriverio/tree/master/packages/webdriver).
+
+```
+let  webdriverio  =  require('webdriverio');
```
[the remark this answers is not in the document]
Comment (ganeshmani) [reply docs/recipes/testing-with-selenium.md]: Got it..Thanks

**Comment by ganeshmani** [comment]
> Also, webdriverio it is Not selenium binding, it implements jsonwp and w3c protocols by itself.

Issue was mentioned with Selenium. i thought that it needs to be implemented with Selenium webdriver. 
@novemberborn Correct me if i am wrong..

**Comment by CrispusDH** [comment]
@ganeshmani what is Selenium Webdriver? Do you meant official Selenium binding for `js`? Here is the link to WebdriverJs - https://github.com/SeleniumHQ/selenium/wiki/WebDriverJs .
`WebdriverIO` - is a framework that implement `jsonwp` and `w3c` protocols by itself. Without using/wrapping Selenium WebdriverJs. 
`Protractor` - is a framework that wraps Selenium WebdriverJs implementation and adding own methods.
`SelenideJS` - is a framework (that was created by several community guys), that implements protocols by itself also without using/wrapping Selenium WebdriveJs.
`Puppeteer` - using DevTools protocol to interact with browser
`Cypress` - hack browser and use just java script
Just to be clear, specify what do you want to cover :)

**Comment by novemberborn** [comment]
Closing due to inactivity. We'd be happy to pick this up again when you have the time to get back to this.

**Pull request closed without merging**